extensions/timeseries/src/main/java/ai/djl/timeseries/timefeature/Lag.java

```diff
@@ -72,7 +106,34 @@ public static List<Integer> getLagsForFreq(String freqStr) {
         return getLagsForFreq(freqStr, 1200);
     }
 
-    private static List<List<Integer>> makeLagsForDay(float multiple) {
+    private static List<List<Integer>> makeLagsForSecond(double multiple) {
```

Comment:
It looks like we can refactor all of these methods into a single method of the form `makeLagsForTimeStep(int numCycles, int multiplier, double multiple)` or something similar. Is there a reason we have multiple private methods for each supported time-step/frequency?

Reply:
I originally thought that writing this would make it more clear to the user which piece of the lags to generate for different frequencies. But now it seems more concise to refactor them into one method.

Reply:
I think that's a fair point, especially if these were public methods. Even if we want to keep all these methods for clarity, I think a good chunk of the body can be refactored out into a separate method since between methods only a couple numbers differ.

Reply:
Yes, that's great. Thanks!